### ISSUE TITLE
Error importing Mongo from '@telegraf/session': Module not found: Default condition should be last one

### ISSUE THREAD

**Issue by TimPchelintsev** [telegraf/telegraf#1810]
There is no GitHub link for `@telegraf/session` package in npm so I hope this is the right place to report.

Thanks for great lib. I tried to import Mongo session store in Next.js api handler and got error.

```javascript
// given "@telegraf/session": "2.0.0-beta.3" in package.json
// Next.js app pages/api/webhooks/telegram.ts
import { Mongo } from '@telegraf/session/mongodb'

// Next.js gives me error: Module not found: Default condition should be last one

```

Removing import line solves the issue and all other code for telegram bot works as expected, with default in-memory session.

**Comment by MKRhere** [comment]
> There is no GitHub link

Thanks, I'll fix that. This is an issue for https://github.com/telegraf/session.

Regardless, did you install the peerDeps as suggested in the documentation?

**Comment by TimPchelintsev** [comment]
Oh I see, it seems I had mongodb driver already installed from another dependency and version was incorrect.
After manual installation import works👍

**Comment by MKRhere** [comment]
Thanks for confirming! Meanwhile I've updated the GitHub repo and issues URL in npm.

**Comment by TimPchelintsev** [comment]
Oh it seems I get this error even after manual install of mongodb dependency.
I tried to create [minimal reproduction on codesandbox](https://codesandbox.io/p/sandbox/immutable-wave-0dcq0k?file=%2Fpackage.json&selection=%5B%7B%22endColumn%22%3A27%2C%22endLineNumber%22%3A7%2C%22startColumn%22%3A27%2C%22startLineNumber%22%3A7%7D%5D)

**Comment by MKRhere** [comment]
Fixed in v2.0.0-beta.6. @vercel / Nextjs didn't need to be this pedantic about "types" after "default" in sub-path exports.
